src/content/en/tools/lighthouse/audits/time-to-interactive.md

```diff
@@ -2,33 +2,69 @@ project_path: /web/tools/_project.yaml
 book_path: /web/tools/_book.yaml
 description: Reference documentation for the "Time to Interactive" Lighthouse audit.
 
-{# wf_updated_on: 2018-07-23 #}
-{# wf_published_on: 2016-10-05 #}
-{# wf_blink_components: N/A #}
+{# wf_updated_on: 2018-08-16 #}
+{# wf_published_on: 2018-08-16 #}
+{# wf_blink_components: Platform>DevTools #}
 
-# Time to Interactive  {: .page-title }
+# Time to Interactive {: .page-title }
 
 ## Overview {: #overview }
 
-Page load is a key aspect of how a user perceives the performance of your
-page. See [Measure Performance with the RAIL Method](/web/fundamentals/performance/rail) for more information.
+<blockquote>
+  <p>
+    Load is not a single moment in time — it’s an experience that no one metric can fully capture.
+    There are multiple moments during the load experience that can affect whether a user perceives
+    it as "fast" or "slow".
+  </p>
+  <p>--- <a class="external" href="https://w3c.github.io/paint-timing/">Paint Timing spec</a></p>
+</blockquote>
 
-This audit identifies the time at which a page appears to be ready enough that
-a user can interact with it.
+The Time to Interactive (TTI) metric measures how long it takes a page to become interactive.
+"Interactive" is defined as the point where:
+
+* The page has displayed useful content, which is measured with [First Meaningful Paint][FMP].
+* Event handlers are registered for most visible page elements.
+* The page responds to user interactions within 50 milliseconds.
+
+[FMP]: /web/tools/lighthouse/audits/first-meaningful-paint
+
+Some sites optimize content visibility at the expense of interactivity. This can create a frustrating
+user experience. The site appears to be ready, but when the user tries to interact with it, nothing happens.
 
 ## Recommendations {: #recommendations }
 
-See [Speed Index](speed-index#recommendations) for more help on improving page load performance.
-The lower your Time to Interactive score, the better.
+To improve your TTI score, defer or remove unnecessary JavaScript work that occurs during page load.
+See [Optimize JavaScript Bootup][bootup] and [Reduce JavaScript Payloads with Tree Shaking][tree shaking],
+and [Reduce JavaScript Payloads with Code Splitting][code splitting].
+
+[bootup]: /web/fundamentals/performance/optimizing-content-efficiency/javascript-startup-optimization/
+[tree shaking]: /web/fundamentals/performance/optimizing-javascript/tree-shaking/
+[code splitting]: /web/fundamentals/performance/optimizing-javascript/code-splitting/
+
+### Tracking TTI in the real world {: #RUM }
+
+To measure when TTI actually occurs on your users' devices, see [Tracking TTI][RUM].
```

Comment:
It might be worth caveating that TTI can be difficult to measure in the wild and that's what [FID](https://developers.google.com/web/updates/2018/05/first-input-delay) is for, might be too much to complicate the story here though :/